qiskit_optimization/algorithms/grover_optimizer.py

```diff
@@ -335,8 +353,14 @@ def __init__(self, x: Union[List[float], np.ndarray], fval: float, variables: Li
                 that is expected to be identical with ``fval``.
             threshold: The threshold of Grover algorithm.
             status: the termination status of the optimization algorithm.
+            samples: the x value, the objective function value of the original problem,
+                the probability, and the status of sampling.
+            raw_samples: the x values of the QUBO, the objective function value of the QUBO,
```

Comment:
`samples` says its the `x value` (not 's' so singular), whereas `raw_samples` says `x values` i.e. plural. Are `samples` and `raw_samples` the same type so that 'raw_samples' should say `x value` just like `samples' does. They both seem to be a List[SolutionSample] so having one as 'x value' and the other as 'x values' seems misleading - or it did to me as the latter seems to be more than one x value, which samples is not.

Reply:
Right, I forgot to put "s" for x value in `samples`. Thanks!